pkg/controllers/termination/suite_test.go

```diff
@@ -89,18 +88,83 @@ var _ = Describe("Termination", func() {
 					v1alpha2.ProvisionerNamespaceLabelKey: "default",
 				},
 			})
-			ExpectCreated(env.Client, node)
-			ExpectCreated(env.Client, test.Pod(test.PodOptions{
+			podEvict := test.Pod(test.PodOptions{
+				NodeName: node.Name,
+			})
+			podSkip := test.Pod(test.PodOptions{
 				NodeName:    node.Name,
 				Tolerations: []v1.Toleration{{Key: v1.TaintNodeUnschedulable, Operator: v1.TolerationOpExists, Effect: v1.TaintEffectNoSchedule}},
-			}))
+			})
+			ExpectCreated(env.Client, node)
+			ExpectCreated(env.Client, podEvict, podSkip)
```

Comment:
This variadic is fancy and can take (env.Client, node, podevict, podskip)

Reply:
True!